app/javascript/map.js

```diff
@@ -0,0 +1,15 @@
+function initMap() {
```

Comment:
The problem with this approach is that you will use this js not only on your map page but on all pages. So please change it to the way that only views with map and you inputs will use this js

Reply:
Ok sorry it seems that you already did it :D. Just rename this file to something more accurate like `map_for_form.js`